docs/api-docs/analytics/bodl-for-storefronts.mdx

```diff
@@ -20,37 +20,103 @@ For scripts to successfully use BODL to access storefront data, the following mu
 ## BODL event schemas
 
 BODL exposes data when shoppers trigger storefront events. Currently, BODL supports the following storefront events:
-- Start checkout
-- Purchase order 
 
-### Start checkout event schema
+- [View product page](#view-product-page-event-schema)
+- [View product category](#view-product-category-event-schema)
+- [Add product to cart](#add-product-to-cart-event-schema)
+- [View cart](#view-cart-event-schema)
+- [Remove product from cart](#remove-product-from-cart-event-schema)
+- [Start checkout](#start-checkout-event-schema)
+- [Purchase order](#purchase-order-event-schema)
 
-When the shopper clicks on the button to initiate a checkout process, the following fields appear in the BODL object:
+### View product category
+
+When the shopper clicks a button to view the category page for a product, the following fields appear in the BODL object for the `categoryViewed` event:
+
+| Event field | Type | Description | BigC data map |
+|:---|:---|:---|:---|
+| `category_id` | integer | ID of the category. | [Get all categories](/docs/rest-management/catalog/category#get-all-categories) <br /> `data.id` |
+| `category_name` | string | Name of the category. | [Get all categories](/docs/rest-management/catalog/category#get-all-categories) <br /> `data.name` |
+| `channel_id` | integer | The BigCommerce channel ID. Channel ID of the channel where checkout is taking place. | [Get all channels](/docs/rest-management/channels#get-all-channels) <br /> `data.id` |
+| `event_id` | string | Unique ID for each event instance. | Unique event ID generated by BigCommerce. |
+| `line_items[]` | array of `line_item` objects | Items in the category. | See [line_item](#line_item-object-schema) object for event fields |
+
+### View product page
+
+When the shopper clicks a button to view a product page, the following fields appear in the BODL object for the `pageViewed` event:
+
+| Event field | Type | Description | BigC data map |
```

Comment:
I know this is a dumb question but I have to ask. Is this approved by marketing? House rules are that we spell BigCommerce out each time, but I could be wrong.

Reply:
I think I just didn't see it, but just changed it to "BigCommerce" everywhere